content/_index.md

```diff
@@ -5,13 +5,29 @@ draft: false
+## About Us
 
-Over the course of the past few weeks, the global community of technologists, privacy experts, and epidemiologists has worked tirelessly towards a secure, privacy-first, GDPR-compliant, and open-source approach to enable globally compatible digital contact tracing. Our goal is to flatten the curve of infections and re-infections in the COVID-19 epidemic. Scientific studies such as [this one from Oxford](https://science.sciencemag.org/content/early/2020/03/30/science.abb6936) have shown that contact tracing can slow the spread of COVID-19 and help safely end lockdowns.
+Founded in April 2020, the TCN Coalition is a global community of technologists united by the mission to support exposure notification apps during the COVID-19 pandemic. The TCN Coalition believes that exposure notification apps must be interoperable, trusted by the public, and privacy-preserving to quickly and safely reduce the need for lockdowns. 
 
-Our first and foremost goal is to get secure tracing apps running on billions of users’ devices globally — fast. The TCN Coalition’s easy-to-implement privacy-first protocol, agreed upon and reviewed by dozens of experts, is open-source, extensible, free of charge, and available for implementation immediately. TCN — the core of the protocol — stands for temporary contact number. This completely anonymous number is generated to privately record interactions between compatible mobile devices without allowing them to be tracked.
+We achieve this mission through several programs: connecting and educating app developers about navigating the public health environment, assisting public health agencies in understanding how to deploy this technology safely and effectively, and supporting exposure notification apps with open source software.
 
-We urge all major technology companies, app developers and governments to implement apps compatible with this shared protocol. We welcome other groups engaged in similar efforts to join us at [TCN-coalition.org](http://TCN-coalition.org). We also encourage any other contact tracing coalitions to adopt similarly secure privacy-first protocols.
+## The Challenge
+
+A large number of organizations have sprung up to either enable or build out exposure notification applications since the start of the Covid-19 pandemic.
+
+These organizat ions span the globe and many of them have overlapping missions and work. On top of that, they are largely focused on their own product and are not thinking about the broader ecosystem, including how interoperability between these systems might work. 
+
+The TCN Coalition serves as a gathering place for the overall effort. We increase visibility across the ecosystem, allowing for better communication, more effective use of resources, and faster delivery of effective exposure notification solutions as a result. 
```

Comment:
"The TCN Coalition serves as a gathering place to increase visibility across the ecosystem, allowing for better communication, more effective use of resources, and faster delivery of effective exposure notification solutions as a result." (I would merge these two sentences)